corehq/apps/adm/models.py

```diff
@@ -348,8 +349,10 @@ def calculate_totals(self, column_data):
     def get_couch_view_data(self, key, datespan=None):
         default_value = None
         try:
-            now = tz_utils.adjust_datetime_to_timezone(getattr(datespan, self.start_or_end or "enddate"),
-                from_tz=datespan.timezone, to_tz=pytz.utc)
+            now = UserTime(
+                getattr(datespan, self.start_or_end or "enddate"),
+                datespan.timezone
+            ).server_time().done().replace(tzinfo=pytz.utc)
```

Comment:
Seems weird to have `done()` in the middle of a chained method call. Is this a special case?